### DISCUSSION TITLE
Does WinGet Have the Option to Change the Install Directory, Especially Universally?

### DISCUSSION THREAD

**Discussion by Foxxsayy** [microsoft/winget-cli#2778] [Q&A] [answered]
I was looking in the documentation and I may have missed it, but can I change the default install directory for WinGet? I want _most_ of my programs installed in a different folder than program files. 

I saw quite a few threads about this, but has it been implemented yet?

**Comment by Karl-WE** [comment]
I wouldn't say so. Unless the installer has such options you could only create own yaml files.

**Reply by cgerke** [reply]
It's 100% dependent on the vendor and their installer.

**Comment by denelon** [comment, marked as answer]
If installers support "location" the "--location" argument supports this behavior.

`winget install -?` displays the majority of switches.
